src/plugins/Breakpoints.ts

```diff
@@ -0,0 +1,50 @@
+import { IAd, IAdBreakpoints } from '../types';
+import AdJsError from '../utils/AdJsError';
+import isBetween from '../utils/isBetween';
+import throttle from '../utils/throttle';
+import GenericPlugin from './GenericPlugin';
+
+class BreakpointsPlugin extends GenericPlugin {
+  public currentConfines: number[] = [];
+  private EVENT_KEY: string = 'resize';
+  private THROTTLE_DURATION: number = 200;
+
+  public beforeCreate() {
+    this.determineCurrentBreakpoint();
```

Comment:
We're missing the check for `refreshOnBreakpoint`.

Documentation here: https://adjs.dev/#/responsive-plugin?id=options